stanford_profile.info.yml

```diff
@@ -44,32 +41,17 @@ install:
   - admin_toolbar_links_access_filter:admin_toolbar_links_access_filter
   - admin_toolbar_tools:admin_toolbar_tools
   - config_split:config_split
-  - profile_split_enable:profile_split_enable
   - allowed_formats:allowed_formats
   - chosen:chosen
-  - coffee:coffee
   - default_content
-  - honeypot:honeypot
   - pathauto:pathauto
   - redirect:redirect
-  - webform:webform
-  - xmlsitemap:xmlsitemap
   - nobots:nobots
 
 dependencies:
   # Stanford functionality.
-  - su-sws:stanford_date_formats
```

Comment:
Are we removing these with the intention to add them back in on the appropriate ticket or what?

Reply:
yeah. this is just very boiler plate install that we'll expand on as we go. i was getting conflicts with these lines. I kinda think its because of the `su-sws` prefix. during install it kept saying "not found". we might have to investigate the prefix later